### ISSUE TITLE
uSplit License

### ISSUE THREAD

**Issue by HarveyWilliams** [EndzoneSoftware/uSplit#2]
Hi Endzone. I'm intrigued by uSplit. I've had a lot of pain setting up new GA experiments for clients and it seems like your package may streamline the process. However, I haven't put the time in to installing it yet.

On the main uSplit page (http://www.happyporch.com/usplit-ab-testing-for-umbraco/) it says that there is a Community version which is free. Is there a Community license too?

On the experiment tutorial page (http://www.happyporch.com/running-ab-experiments-like-pro-usplit/), it says:

> While the free version of the plugin supports all the features, the duration for which each experiment might run is capped at 3 days, which is usually not enough for a winner to be chosen, unless you have a very high volume of traffic.

Can we license is with the Community version, or do we need to pay in order to unrestrict the 3 day cap?

Thanks.

**Comment by ondrejpialek** [comment]
Hi Harvey!

Thanks for the inquiry. uSplit is free. The packages offered on our website are commercial support contracts for enterprise customers.

The licenses you refer to are obsolete. We initial thought we'd license uSplit itself, and we took the screenshots and wrote those supplementary materials back then. Since then we released uSplit open source and the licensing options are no longer there. Just grab it from NuGet and feel free to give it a go.

Only if you had a critical site you wanted to support, or are a Digital Agency with many sites should you consider buying one of our support options. But so long as you are happy doing the work yourself you don't have to obtain any license ;)

Hope this answered your question.
Cheers,
Ondrej.

PS: there is a demo of uSplit I made for uHangout: https://www.youtube.com/watch?v=WQysVNLyQM8
